models/spring-ai-openai/src/main/java/org/springframework/ai/openai/api/OpenAiModerationApi.java

```diff
@@ -63,31 +68,47 @@ public class OpenAiModerationApi {
 	 */
 	public OpenAiModerationApi(String baseUrl, ApiKey apiKey, MultiValueMap<String, String> headers,
 			RestClient.Builder restClientBuilder, ResponseErrorHandler responseErrorHandler) {
+		this(baseUrl, OpenAiApiConstants.DEFAULT_MODERATION_PATH, apiKey, headers, restClientBuilder,
+				responseErrorHandler);
+	}
 
+	/**
+	 * Create a new OpenAI Moderation API with the provided base URL.
+	 * @param baseUrl the base URL for the OpenAI API.
+	 * @param apiKey OpenAI apiKey.
+	 * @param restClientBuilder the rest client builder to use.
+	 * @param moderationPath the moderation path to use.
+	 */
+	public OpenAiModerationApi(String baseUrl, String moderationPath, ApiKey apiKey,
+			MultiValueMap<String, String> headers, RestClient.Builder restClientBuilder,
+			ResponseErrorHandler responseErrorHandler) {
```

Comment:
Unifying the variable order in OpenAiApi could improve consistency.
The suggested order is: `baseUrl` → `apiKey` → `headers` → `path`

For example:
```suggestion
	public OpenAiModerationApi(String baseUrl, ApiKey apiKey, MultiValueMap<String, String> headers,
	String moderationPath, RestClient.Builder restClientBuilder,
	ResponseErrorHandler responseErrorHandler) {
```

Reply:
Your suggestion is great, I'll optimize it, thanks